src/main/java/com/webservices/company/repository/CompanyRepository.java

```diff
@@ -48,7 +49,12 @@ public Company add(Company companyToBeAdded) {
 
     @Override
     public Company get(Long id) {
-        Company company = jdbcTemplate.queryForObject("select * from companies where id = " + id, rowMapper);
+        Company company = null;
+        try {
+            company = jdbcTemplate.queryForObject("select * from companies where id =  " + id, rowMapper);
```

Comment:
it is a small misstake but it is better to to like `jdbcTemplate.queryForObject("select * from companies where id = ?", rowMapper, id)`